notebooks/notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !bash "../infrastructure/downloader.sh" &gt; "../outputs/logs/download_data.txt"
</pre><p>I suggest that you describe the OS you used to run the analysis and provide some alternatives to download the data on other OS such as Windows. alternatives are libraries such as requests, urllib or pooch.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/repro-challenge-team-3/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>This has been done</p>

Reply:
closing conversation, resolved